### DISCUSSION TITLE
云函数能获取到客户端的 ip 吗？

### DISCUSSION THREAD

**Discussion by yenche123** [labring/laf#285] [Feedbacks] [answered]
如题。我打印过 `FunctionContext.headers` 没看到 ip，还是说挂在 `process` 上而我不知道？

**Comment by maslow** [comment]
我记得是可以的，不过我今天也试了，确实没有，明天我检查一下 路由配置，有没有转发逻辑。

**Comment by maslow** [comment]
'x-real-ip': '125.120.107.22'

lafyun 上我已经把 proxy ip 规则加上了。

**Reply by yenche123** [reply]
好的，感谢！

**Comment by maslow** [comment, marked as answer]
```conf
proxy_set_header Host $host;
proxy_set_header X-Real-IP $remote_addr;
```

**Comment by yenche123** [comment]
必须夸一下：这个工单响应比任何大厂都快了！我不允许还有人不知道 `Laf` !!
